recipes/openms/meta.yaml

```diff
@@ -22,9 +22,9 @@ requirements:
     - python
     - zlib
     - libtool
-    - xerces-c
+    - xerces-c=3.1
```

Comment:
I guess you have to specify via bool '=='

Reply:
thx

Reply:
ok, this was alright - sry!